modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -31,6 +31,9 @@
             v-model="checkedProxy"
           />
         </div>
+
+        <slot name="actions" />
```

Comment:
Ok this strategy won't be enough because we will need the actual data of the current template we clicked on in order to open the right editor manager or to redirect to the piece _url if this one exists.

Reply:
So a good solution would be to pass an array of buttons, each one would have an action name associated with it, and the card component will be able to emit the right event based on that action name, like that the parent component can listen to those events and know on which button the user clicked on.